packages/core/contracts-ovm/insured-bridge/implementation/BridgeAdmin.sol

```diff
@@ -118,6 +118,11 @@ contract BridgeAdmin is BridgeAdminInterface, Ownable, OVM_CrossDomainEnabled {
      *        CROSSDOMAIN ADMIN FUNCTIONS             *
      **************************************************/
 
+    // TODO: In following functions, we need to consider two things concerning asynchronous cross-domain messaging:
```

Comment:
+1

For now I think both of these can be left up to the admin (they put their own gas and deal with reverting on L2). As long as users don't have to specify this ever it should be fine.

Reply:
There are no cross-domain functions that users can call right?

Reply:
nope. not rn